dashboard/src/reducers/namespace.test.ts

```diff
@@ -42,13 +43,22 @@ describe("namespaceReducer", () => {
   context("when ERROR_NAMESPACE", () => {
     const err = new Error("Bang!");
 
-    it("leaves namespaces intact and sets error", () => {
+    it("when listing leaves namespaces intact and but ignores the error", () => {
```

Comment:
FWIW, I think this inconsistency will go away if and when we're able to list the namespaces to which the user has access.

Reply:
yep